third_party/flann.BUILD

```diff
@@ -0,0 +1,164 @@
+    "@boost//:array",
+    "@boost//:asio",
+    "@boost//:assert",
+    "@boost//:atomic",
+    "@boost//:beast",
+    "@boost//:bimap",
+    "@boost//:bind",
+    "@boost//:call_traits",
+    "@boost//:callable_traits",
+    "@boost//:cerrno",
+    "@boost//:checked_delete",
+    "@boost//:chrono",
+    "@boost//:circular_buffer",
+    # "@boost//:compute",
+    "@boost//:concept",
+    "@boost//:concept_archetype",
+    "@boost//:concept_check",
+    "@boost//:config",
+    "@boost//:container",
+    # "@boost//:context",
+    "@boost//:conversion",
+    "@boost//:core",
+    # "@boost//:coroutine",
+    "@boost//:cstdint",
+    "@boost//:current_function",
+    "@boost//:date_time",
+    "@boost//:detail",
+    "@boost//:dynamic_bitset",
+    "@boost//:enable_shared_from_this",
+    # "@boost//:endian",
+    "@boost//:exception",
+    "@boost//:exception_ptr",
+    # "@boost//:fiber",
+    "@boost//:filesystem",
+    "@boost//:foreach",
+    "@boost//:format",
+    "@boost//:function",
+    "@boost//:function_types",
+    "@boost//:functional",
+    "@boost//:fusion",
+    "@boost//:get_pointer",
+    "@boost//:heap",
+    # "@boost//:icl",
+    "@boost//:integer",
+    "@boost//:interprocess",
+    "@boost//:intrusive",
+    "@boost//:intrusive_ptr",
+    "@boost//:io",
+    "@boost//:iostreams",
+    "@boost//:is_placeholder",
+    "@boost//:iterator",
+    "@boost//:lexical_cast",
+    "@boost//:limits",
+    "@boost//:math",
+    "@boost//:mem_fn",
+    "@boost//:move",
+    "@boost//:mp11",
+    "@boost//:mpl",
+    "@boost//:multi_index",
+    # "@boost//:multiprecision",
+    "@boost//:noncopyable",
+    "@boost//:none",
+    # "@boost//:numeric",
+    "@boost//:numeric_conversion",
+    "@boost//:numeric_ublas",
+    "@boost//:operators",
+    "@boost//:optional",
+    "@boost//:parameter",
+    # "@boost//:pool",
+    "@boost//:predef",
+    "@boost//:preprocessor",
+    "@boost//:process",
+    "@boost//:program_options",
+    "@boost//:property_tree",
+    "@boost//:ptr_container",
+    "@boost//:random",
+    "@boost//:range",
+    "@boost//:ratio",
+    # "@boost//:rational",
+    "@boost//:ref",
+    "@boost//:regex",
+    "@boost//:scope_exit",
+    "@boost//:scoped_array",
+    "@boost//:scoped_ptr",
+    "@boost//:serialization",
+    "@boost//:shared_array",
+    "@boost//:shared_ptr",
+    "@boost//:signals2",
+    "@boost//:smart_ptr",
+    "@boost//:spirit",
+    "@boost//:static_assert",
+    "@boost//:swap",
+    "@boost//:system",
+    "@boost//:thread",
+    "@boost//:throw_exception",
+    "@boost//:timer",
+    "@boost//:tokenizer",
+    # "@boost//:tribool",
+    "@boost//:tuple",
+    "@boost//:type",
+    "@boost//:type_index",
+    "@boost//:type_traits",
+    "@boost//:typeof",
+    "@boost//:unordered",
+    "@boost//:utility",
+    "@boost//:uuid",
+    "@boost//:variant",
+    "@boost//:version",
+    "@boost//:visit_each",
+]
+
+FLANN_DEPS = BOOST_TARGETS + [ "@lz4//:lz4" ] #"@hdf5//:hdf5",
```

Comment:
Pro tip: When you are using the default target, you can just leave off the target name. In other words, if the target has the same name as the directory it is in, you can leave it off. In this case, you can just do `@lz4` instead of `@lz4//:lz4`. It's the same thing.